mlir/include/mlir/Conversion/RockPasses.td

```diff
@@ -1,35 +1,35 @@
-//===-- MIOpenPasses.td - Conversion pass definition file --*- tablegen -*-===//
+//===-- RockPasses.td - Conversion pass definition file --*- tablegen -*-===//
```

Comment:
Let's make this file rocMLIRPasses.td? Since I'm pretty sure it also has MIGraphXToTosa